site/docs/v1/tech/core-concepts/applications.adoc

```diff
@@ -411,6 +411,9 @@ The form that will be used in the FusionAuth UI for adding and editing user regi
 [field]#User self-service# [optional]#Optional# [since]#Available since 1.26.0#::
 The form that will be used with the hosted login pages for user self-service account management.
 
+[field]#Require current password# [optional]#Optional# [since]#Available since 1.45.0#::
+If set, a user will have to provide their current password when changing their password on the self-service account form. Note that a User self-service form must be selected for this setting to have any effect.
```

Comment:
We should also mention that this only applies when the form has a password field configured.